src/test/java/bio/terra/service/filedata/google/firestore/ArrayMultiFileLoadTest.java

```diff
@@ -143,10 +143,8 @@ public void arrayMultiFileLoadSuccessTest() throws Exception {
 
     assertThat(
         "getting load history has the same items as response from bulk file load",
-        loadHistoryList.getItems().stream()
-            .map(TestUtils::toBulkLoadFileResultModel)
-            .collect(Collectors.toSet()),
-        Matchers.equalTo(Set.copyOf(result.getLoadFileResults())));
+        loadHistoryList.getItems().stream().map(TestUtils::toBulkLoadFileResultModel).toList(),
+        containsInAnyOrder(result.getLoadFileResults().toArray()));
```

Comment:
I think `toArray()` is unnecessary here since `containsInAnyOrder()` can also accept `Collection` as well as `T[]`.

Reply:
Oh cool!  I don't know how I missed that.

Reply:
Actually, I am having a hard time getting this to work. `containsInAnyOrder` can accept a `Collection` of `Matchers`, not a `Collection<T>`.  Let me know if you think I'm missing something.

Reply:
You're right, I missed the `Collection<Matcher<? super T>>` type restriction. I was sure there was another way to do this but I guess this is the best way.